pom.xml

```diff
@@ -353,8 +350,7 @@
             <configuration>
               <fork>true</fork>
               <compilerArgs combine.children="append">
-                <arg>-XDcompilePolicy=simple</arg>
```

Comment:
This disables errorprone, again, not acceptable, errorprone must be active when this profile is activated.

Reply:
[see comment](https://github.com/geotools/geotools/blob/e66fb740d4e05e6d20d2c942d59815068cc570f8/pom.xml#L353)